### PR TITLE
fix some typos in apiserver

### PR DESCRIPTION
**Pull request by supereagle**
**What this PR does / why we need it**:

**Which issue this PR fixes**: 

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by k82cn** [comment]
@k8s-bot ok to test

**Comment by cblecker** [comment]
Looks like an unrelated flake.
@k8s-bot unit test this

**Comment by cblecker** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by cblecker** [comment]
/assign @smarterclayton 
So it shows up in his PR dashboard as pending approval :)

**Comment by supereagle** [comment]
@smarterclayton  This PR needs your approval :). Thanks.

**Comment by cblecker** [comment]
@k8s-bot verify test this

File: staging/src/k8s.io/apiserver/pkg/endpoints/installer.go [hunk 1]
```diff
@@ -50,13 +50,13 @@ type APIInstaller struct {
...
-// Struct capturing information about an action ("GET", "POST", "WATCH", PROXY", etc).
+// Struct capturing information about an action ("GET", "POST", "WATCH", "PROXY", etc).
 type action struct {
-	Verb          string               // Verb identifying the action ("GET", "POST", "WATCH", PROXY", etc).
+	Verb          string               // Verb identifying the action ("GET", "POST", "WATCH", "PROXY", etc).
 	Path          string               // The path of the action
 	Params        []*restful.Parameter // List of parameters associated with the action.
 	Namer         handlers.ScopeNamer
-	AllNamespaces bool // true iff the action is namespaced but works on aggregate result for all namespaces
+	AllNamespaces bool // true if the action is namespaced but works on aggregate result for all namespaces
```
Comment (smarterclayton) [inline staging/src/k8s.io/apiserver/pkg/endpoints/installer.go]: This is intentional - 'if and only if'

Comment (supereagle) [inline staging/src/k8s.io/apiserver/pkg/endpoints/installer.go]: Thanks. I will revert this.

**Comment by supereagle** [comment]
@smarterclayton   Have updated.

**Pull request merged**